tests/e2e/performance/test_pvc_multi_snapshot_performance.py

```diff
@@ -0,0 +1,156 @@
+import logging
+
+from ocs_ci.ocs import constants, exceptions
+from ocs_ci.ocs.cluster import CephCluster
+from ocs_ci.framework.testlib import (
+    skipif_ocs_version, skipif_ocp_version, E2ETest, performance
+)
+from ocs_ci.ocs.resources import pvc
+from tests import helpers
+
+log = logging.getLogger(__name__)
+
+
+@performance
+@skipif_ocp_version('<4.6')
+@skipif_ocs_version('<4.6')
+class TestPvcMultiSnapshotPerformance(E2ETest):
+    """
+    Tests to measure PVC snapshots creation performance & scale
+    The test is trying to to take the maximum number of snapshot for one PVC
+    """
+
+    def test_pvc_multiple_snapshot_performance(
+        self, interface_iterate, teardown_factory, storageclass_factory,
+        pvc_factory, pod_factory
+    ):
+        """
+        1. Creating PVC
+           size is depend on storage capacity, but not less then 1 GiB
+           it will use ~75% capacity of the Storage, Min storage capacity 1 TiB
+        2. Fill the PVC with 80% of data
+        3. Take a snapshot of the PVC and measure the time of creation.
+        4. re-write the data on the PVC
+        5. Take a snapshot of the PVC and measure the time of creation.
+        6. repeat steps 4-5 the numbers of snapshot we want to take : 512
+        7. print all information.
+
+        Raise:
+            StorageNotSufficientException: in case of not enough capacity
+
+        """
+        create_times = []  # list for all snapshots results
+        num_of_snaps = 512  # number of snapshots to take
+
+        # Getting the total Storage capacity
+        ceph_cluster = CephCluster()
+        ceph_capacity = int(ceph_cluster.get_ceph_capacity())
+
+        # Use 70% of the storage capacity in the test
+        capacity_2_use = int(ceph_capacity * 0.7)
+
+        # since we do not want to use more then 75%, we add 35% to the needed
```

Comment:
shouldn't we add 25% and not 35?

Reply:
the typo is in the comment, changed to 65%.